src/main/java/io/ballerina/plugins/idea/runconfig/application/BallerinaApplicationRunConfiguration.java

```diff
@@ -46,15 +49,28 @@ protected BallerinaApplicationRunConfiguration(Project project, ConfigurationFac
     public @Nullable RunProfileState getState(@NotNull Executor executor,
                                               @NotNull ExecutionEnvironment executionEnvironment)
             throws ExecutionException {
-        if (Objects.equals(BallerinaSdkService.getInstance().getBallerinaVersion(executionEnvironment.getProject()),
-                "")) {
+        if (BallerinaSdkService.getInstance().getBallerinaVersion(executionEnvironment.getProject()).isEmpty()) {
             BallerinaPluginNotifier.notifyBallerinaNotDetected(executionEnvironment.getProject());
             return null;
         }
 
+        String script = getOptions().getScriptName();
+
+        try {
+            Path path = Paths.get(script).normalize();
+            if (!Files.exists(path)) {
+                throw new IllegalArgumentException("File does not exist");
+            }
+            script = path.toString();
+        } catch (Exception e) {
+            BallerinaPluginNotifier.customNotification(executionEnvironment.getProject(), NotificationType.ERROR,
+                    "Invalid script", "Provided script path is not valid or does not exist.");
```

Comment:
What do we mean by "script" in here? Is it a Ballerina standalone file?

Reply:
It is the bal file or package that we set in the run configuration

Reply:
In that case, the below would look less confusing to the users. WDYT?
```ballerina
BallerinaPluginNotifier.customNotification(executionEnvironment.getProject(), NotificationType.ERROR,
                    "Invalid Path", "Provided path is not valid or does not exist.");
```